### PR TITLE
MM-28817: README.md as a landing page

### PR DESCRIPTION
**Pull request by agarciamontoro**
#### Summary

Please, review both the structure and the language. Feel free to dissect it and suggest all the changes you consider needed!

I added everyone in case you wanna take a look, but I'd love a thorough review by @justinegeffen and @itao in particular!

A direct link to the rendered file: https://github.com/mattermost/mattermost-plugin-incident-response/blob/MM-28817.readme/README.md

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28817

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,145 @@
...
+Read more about the Backstage here.
+
+## Contributing
+
+### Prerequisites
+
+There are two things you need to do before starting development on the plugin:
+
+1.  You will need a running server to deploy the plugin to, and the web app to manually test your changes. If you have never set up your Mattermost development environment, do so following the instructions for both the [server](https://developers.mattermost.com/contribute/server/developer-setup/) and the [web app](https://developers.mattermost.com/contribute/webapp/developer-setup/).
+2.  The plugin needs a Mattermost E20 License to start. Alternatively, you can run it for development by configuring your Mattermost server to enable both testing and developer mode. You can do so directly on the server's `conf/conf.json` file, setting both `ServiceSettings.EnableTesting` and `ServiceSettings.EnableDeveloper` to `true` (don't forget to restart the server afterwards). You can also configure these settings via the System Console, under Environment > Developer. See [License](#License) for more information.
```
Comment (lieut-data) [inline README.md:55]:
```suggestion
 2.  The plugin needs a Mattermost E20 License to start. Alternatively, you can run it for development by configuring your Mattermost server to enable both testing and developer mode. You can do so directly on the server's `config/config.json` file, setting both `ServiceSettings.EnableTesting` and `ServiceSettings.EnableDeveloper` to `true` (don't forget to restart the server afterwards). You can also configure these settings via the System Console, under Environment > Developer. See [License](#License) for more information. 
```

File: README.md [hunk 2]
```diff
@@ -1,28 +1,145 @@
...
+```
+
+The default target for `make` triggers the commands `check-style`, `test` and `dist`: if everything works as expected, you will end up with a packaged plugin in the `dist/` directory, ready to be uploaded to Mattermost via the System Console. There are more `make` commands: run `make help` to get a comprehensive list of them.
 
-##### Running E2E Tests:
+One particularly interesting command is `make deploy`, which looks for a running Mattermost server, and directly deploys the plugin to it and enables it via the API. For this to work, you need to set up a few environment variables before:
 
-_To run the Cypress E2E tests in your local environment, do the following:_
+```sh
+# This is the default URL for a local Mattermost server.
```
Comment (lieut-data) [inline README.md:75]: If we sync with the starter-template, we can get the new power to deploy via the socket which in turn requires no authentication or even configuration. Maybe worth queuing up as a ticket.

**Review by lieut-data — APPROVED** [review]
Incredible work, @agarciamontoro! I'll defer to @justinegeffen for any polish, but it reads extremely well already. Two minor technical comments below.

File: README.md (hunk 2 above)
Comment (agarciamontoro) [reply README.md:75]: A Help Wanted candidate? :)

File: README.md (hunk 1 above)
Comment (agarciamontoro) [reply README.md:55]: Oh great catch, I wrote that from memory and did not check it was right. Thanks! :)

File: README.md [hunk 3]
```diff
@@ -1,28 +1,145 @@
...
+
+There will come a time where you will have several incidents, some of them ongoing, some ended. If you ever need a quick glance at the status of an incident, follow these steps:
+
+1. Navigate to Playbooks & Incidents in the main menu.
+2. Click on Incidents on the header.
+3. See the list of all incidents. You can filter by Commander, Status (ongoing or ended) and look for a specific incident using the search box.
+4. Click on the incident you are interested in.
+5. A summary with some statistics of the incident will show. From this screen, you can also jump to the channel (with the button in the Messages box) and export the channel log, which will download a CSV file with a transcript of all the messages in the channel.
+
+Read more about the Backstage here.
```
Comment (crspeller) [inline README.md:46]:
```suggestion

```

File: README.md [hunk 4]
```diff
@@ -1,28 +1,145 @@
...
+
+## Contributing
+
+### Prerequisites
+
+There are two things you need to do before starting development on the plugin:
+
+1.  You will need a running server to deploy the plugin to, and the web app to manually test your changes. If you have never set up your Mattermost development environment, do so following the instructions for both the [server](https://developers.mattermost.com/contribute/server/developer-setup/) and the [web app](https://developers.mattermost.com/contribute/webapp/developer-setup/).
+ 2.  The plugin needs a Mattermost E20 License to start. Alternatively, you can run it for development by configuring your Mattermost server to enable both testing and developer mode. You can do so directly on the server's `config/config.json` file, setting both `ServiceSettings.EnableTesting` and `ServiceSettings.EnableDeveloper` to `true` (don't forget to restart the server afterwards). You can also configure these settings via the System Console, under Environment > Developer. See [License](#License) for more information.
```
Comment (crspeller) [inline README.md:55]:
```suggestion
 2.  Normally plugin needs a Mattermost E20 License to start. However, you can bypass this and run it for development by configuring your Mattermost server to enable both testing and developer mode. You can do so directly on the server's `config/config.json` file, setting both `ServiceSettings.EnableTesting` and `ServiceSettings.EnableDeveloper` to `true` (don't forget to restart the server afterwards). You can also configure these settings via the System Console, under Environment > Developer. See [License](#License) for more information. 
```

File: README.md [hunk 5]
```diff
@@ -1,28 +1,145 @@
...
+Now you are ready to run the tests from within `tests-e2e/`. There are two alternative ways:
+
+1. run `npm test` to run all tests headlessly.
+2. run `npm run cypress:open` to open the Cypress dashboard, select the test spec you want and run it.
+
+For a full documentation on E2E tests for Mattermost web app and troubleshooting, please see https://developers.mattermost.com/contribute/webapp/end-to-end-tests/.
+
+## License
+
+This repository is licensed under the [Mattermost Source Available License](LICENSE) and requires a valid Enterprise E20 license. See [frequently asked questions](https://docs.mattermost.com/overview/faq.html#mattermost-source-available-license) to learn more.
```
Comment (crspeller) [inline README.md:141]: We should probably put this at the top to not disappoint people.

**Review by crspeller — APPROVED** [review]
Awesome! Non blocking suggestions.

File: README.md (hunk 2 above)
Comment (lieut-data) [reply README.md:75]: Filed as https://mattermost.atlassian.net/browse/MM-29126

File: README.md (hunk 5 above)
Comment (cpoile) [reply README.md:141]: Maybe change to "and requires a valid Enterprise E20 license when used for production."?

File: README.md (hunk 4 above)
Comment (cpoile) [reply README.md:55]: Normally /the/ plugin

**Review by cpoile — APPROVED** [review]
Nice!  🎉

File: README.md (hunk 3 above)
Comment (agarciamontoro) [reply README.md:46]: Oops, this was meant to add a link to the documentation, but I forgot to add it. Anyways, there is no explicit Backstage section in the docs, so I'll remove it as suggested. Thanks for the catch!

File: README.md (hunk 4 above)
Comment (agarciamontoro) [reply README.md:54]: I like that much better, thank you!!!

Comment (agarciamontoro) [reply README.md]: Done in https://github.com/mattermost/mattermost-plugin-incident-response/pull/303/commits/de5f47dcfd0034f137d21b2c3f8182f7b58abac0.

File: README.md (hunk 5 above)
Comment (agarciamontoro) [reply README.md]: Moved to the top, just below the screenshot. I added a link to the Contributing section at the end, too. https://github.com/mattermost/mattermost-plugin-incident-response/pull/303/commits/2c2218256c7f5fa82064fc4936aee1f980661f12

File: README.md [hunk 6]
```diff
@@ -1,28 +1,143 @@
 # Mattermost Incident Response Plugin
 
-This plugin allows you to coordinate and manage incidents within Mattermost.
+[![Build Status](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-incident-response/master.svg)](https://circleci.com/gh/mattermost/mattermost-plugin-incident-response)
+[![Code Coverage](https://img.shields.io/codecov/c/github/mattermost/mattermost-plugin-incident-response/master.svg)](https://codecov.io/gh/mattermost/mattermost-plugin-incident-response)
+[![Release](https://img.shields.io/github/v/release/mattermost/mattermost-plugin-incident-response)](https://github.com/mattermost/mattermost-plugin-incident-response/releases/latest)
+
+Incident Response allows your team to coordinate, manage and resolve incidents from within Mattermost.
```
Comment (justinegeffen) [inline README.md:7]:
```suggestion
Incident Response allows your team to coordinate, manage, and resolve incidents from within Mattermost.
```

Comment (agarciamontoro) [reply README.md:7]: Oh the Oxford comma, always so slippery!

**Review by prapti — APPROVED** [review]
Looks great!! 👍

File: README.md [hunk 7]
```diff
@@ -1,28 +1,143 @@
...
+[![Release](https://img.shields.io/github/v/release/mattermost/mattermost-plugin-incident-response)](https://github.com/mattermost/mattermost-plugin-incident-response/releases/latest)
+
+Incident Response allows your team to coordinate, manage, and resolve incidents from within Mattermost.
+
+![Incident Response plugin screenshot](assets/center_channel.png)
 
 ## License
 
-This repository is licensed under the [Mattermost Source Available License](LICENSE) and requires a valid Enterprise E20 license. See [frequently asked questions](https://docs.mattermost.com/overview/faq.html#mattermost-source-available-license) to learn more.
+This repository is licensed under the [Mattermost Source Available License](LICENSE) and requires a valid Enterprise E20 license when used for production. See [frequently asked questions](https://docs.mattermost.com/overview/faq.html#mattermost-source-available-license) to learn more.
```
Comment (justinegeffen) [inline README.md:13]: @agarciamontoro, 
```suggestion
This repository is licensed under the [Mattermost Source Available License](https://docs.mattermost.com/overview/faq.html#mattermost-source-available-license) and requires a valid Enterprise E20 license when used for production. See [frequently asked questions](https://docs.mattermost.com/overview/faq.html#mattermost-source-available-license) to learn more.
```

File: README.md [hunk 8]
```diff
@@ -1,28 +1,143 @@
...
+
+Although a valid Mattermost Enterprise E20 license is required if using this plugin in production, the [Mattermost Source Available License](LICENSE) allows you to compile and test this plugin in development and testing environments without a Mattermost Enterprise E20 license. As such, we welcome community contributions to this plugin.
```
Comment (justinegeffen) [inline README.md:15]:
```suggestion
Although a valid Mattermost Enterprise Edition E20 license is required if using this plugin in production, the [Mattermost Source Available License](https://docs.mattermost.com/overview/faq.html#mattermost-source-available-license) allows you to compile and test this plugin in development and testing environments without a Mattermost Enterprise Edition E20 license. As such, we welcome community contributions to this plugin.
```

File: README.md [hunk 9]
```diff
@@ -1,28 +1,143 @@
...
+
+On startup, the plugin checks for a valid Mattermost Enterprise E20 license. If you're running the Enterprise Edition of the Mattermost server and don't already have a valid license, you can obtain a trial license from **System Console > Edition and License**. If you're running the Team Edition of the Mattermost server, including when you run the server directly from source, you may instead configure your server to enable both testing (`ServiceSettings.EnableTesting`) and developer mode (`ServiceSettings.EnableDeveloper`). These settings are not recommended in production environments. See [Contributing](#contributing) to learn more about how to set up your development environment.
```
Comment (justinegeffen) [inline README.md:17]:
```suggestion
On startup, the plugin checks for a valid Mattermost Enterprise Edition E20 license. If you're running an Enterprise Edition of Mattermost and don't already have a valid license, you can obtain a trial license from **System Console > Edition and License**. If you're running the Team Edition of Mattermost, including when you run the server directly from source, you may instead configure your server to enable both testing (`ServiceSettings.EnableTesting`) and developer mode (`ServiceSettings.EnableDeveloper`). These settings are not recommended in production environments. See [Contributing](#contributing) to learn more about how to set up your development environment.
```

File: README.md [hunk 10]
```diff
@@ -1,28 +1,143 @@
...
+
+## Features
+
+### Plan your incidents
+
+When an incident starts, you want to focus on triaging and solving the problem as soon as possible. Planning your incidents ahead of time with Playbooks is the best way to do it:
```
Comment (justinegeffen) [inline README.md:23]:
```suggestion
When an incident starts, you want to focus on triaging and solving the problem as soon as possible. Planning your incidents ahead of time with playbooks is the best way to do it:
```

File: README.md (hunk 7 above)
Comment (agarciamontoro) [reply README.md:13]: I think it's good to have the link to the LICENSE file in the repo, as we're linking to the FAQ later. What do you think?

File: README.md (hunk 8 above)
Comment (agarciamontoro) [reply README.md:15]: Same as before.

File: README.md (hunk 7 above)
Comment (lieut-data) [reply README.md:13]: Yeah, let's leave this as-is: the current text was a specific request from @jasonblais in https://github.com/mattermost/mattermost-plugin-incident-response/pull/271/files.

Comment (justinegeffen) [reply README.md:13]: Gotcha, thanks for the clarification. I've updated the suggestion and reverted the license bit.

File: README.md [hunk 11]
```diff
@@ -1,28 +1,143 @@
...
+
+On startup, the plugin checks for a valid Mattermost Enterprise Edition E20 license. If you're running an Enterprise Edition of Mattermost and don't already have a valid license, you can obtain a trial license from **System Console > Edition and License**. If you're running the Team Edition of Mattermost, including when you run the server directly from source, you may instead configure your server to enable both testing (`ServiceSettings.EnableTesting`) and developer mode (`ServiceSettings.EnableDeveloper`). These settings are not recommended in production environments. See [Contributing](#contributing) to learn more about how to set up your development environment.
+
+## Features
+
+### Plan your incidents
+
+When an incident starts, you want to focus on triaging and solving the problem as soon as possible. Planning your incidents ahead of time with playbooks is the best way to do it:
```
Comment (justinegeffen) [inline README.md:23]:
```suggestion
During an active incident, you want to focus on triaging and solving the problem as soon as possible. Planning your incident support strategy ahead of time with playbooks is the best way to ensure incidents run smoothly.
```

Comment (agarciamontoro) [reply README.md:23]: Much better, thank you!

**Comment by lieut-data** [comment]
@agarciamontoro, this looks really great! @justinegeffen, any final changes you would recommend before merging?

**Comment by justinegeffen** [comment]
This is looking awesome, @agarciamontoro!

One point, about the "Development" section: We're changing all the repos to have this information:
"This plugin contains both a server and web app portion. Read our documentation about the Developer Workflow and Developer Setup for more information about developing and extending plugins." with links to https://developers.mattermost.com/extend/plugins/developer-workflow/ and https://developers.mattermost.com/extend/plugins/developer-setup/ respectively. 

The section in this doc is more extensive than the other repos' though so was wondering your thoughts around whether to have that content and those links added?

File: README.md [hunk 12]
```diff
@@ -1,28 +1,143 @@
...
+
+On startup, the plugin checks for a valid Mattermost Enterprise Edition E20 license. If you're running an Enterprise Edition of Mattermost and don't already have a valid license, you can obtain a trial license from **System Console > Edition and License**. If you're running the Team Edition of Mattermost, including when you run the server directly from source, you may instead configure your server to enable both testing (`ServiceSettings.EnableTesting`) and developer mode (`ServiceSettings.EnableDeveloper`). These settings are not recommended in production environments. See [Contributing](#contributing) to learn more about how to set up your development environment.
+
+## Features
+
+### Plan your incidents
+
+During an active incident, you want to focus on triaging and solving the problem as soon as possible. Planning your incident support strategy ahead of time with playbooks is the best way to ensure incidents run smoothly.
+
+1. Navigate to Playbooks & Incidents in the main menu.
```
Comment (justinegeffen) [inline README.md:25]:
```suggestion
1. Navigate to **Main Menu > Incidents & Playbooks**.
```

File: README.md [hunk 13]
```diff
@@ -1,28 +1,143 @@
...
+2. Create a new playbook, either from a predefined template or from scratch. You can add as many stages and tasks as you need. Share the playbook with the people that should decide on your processes, and select the type of channel created when an incident starts: public or private.
```
Comment (justinegeffen) [inline README.md:26]:
```suggestion
2. Create a new playbook, using either a predefined template or from scratch. You can add as many stages and tasks as you need. Share the playbook with the relevant team members, and select the type of channel created when an incident starts - either Public or Private.
```

File: README.md [hunk 14]
```diff
@@ -1,28 +1,143 @@
...
+
+A playbook is a recipe for dealing with and resolving an incident. In a playbook, you can plan ahead so that during an incident responders know exactly what to do. Make sure to schedule a post-mortem analysis to iterate on the design of your playbooks after the incident finishes. Read more about Playbooks [here](https://docs.mattermost.com/administration/devops-command-center.html#creating-a-playbook).
+
+1. Navigate to **Main Menu > Incidents & Playbooks**.
+2. Create a new playbook, using either a predefined template or from scratch. You can add as many stages and tasks as you need. Share the playbook with the relevant team members, and select the type of channel created when an incident starts - either Public or Private.
+3. Save the playbook.
+
+### Start an incident
+
+A server in your cluster is down, a customer is unable to login or your team has detected a performance degradation. Whatever the problem is, if it needs coordination and a quick response, starting an incident will help you focus on solving it. There are two alternative ways to start an incident:
```
Comment (justinegeffen) [inline README.md:33]:
```suggestion
A server in your cluster is down, a customer is unable to log in, or your team has detected a performance degradation. 

Whatever the problem is, if it needs coordination and a quick response, starting an incident will help you focus on solving it. There are two ways to start an incident:
```

**Comment by agarciamontoro** [comment]
> One point, about the "Development" section: We're changing all the repos to have this information:
"This plugin contains both a server and web app portion. Read our documentation about the Developer Workflow and Developer Setup for more information about developing and extending plugins." with links to https://developers.mattermost.com/extend/plugins/developer-workflow/ and https://developers.mattermost.com/extend/plugins/developer-setup/ respectively.
The section in this doc is more extensive than the other repos' though so was wondering your thoughts around whether to have that content and those links added?

At the end of the Prerequisites section there is this:

> If this is the first time you contribute to a Mattermost plugin, the [plugin documentation](https://developers.mattermost.com/extend/plugins/) is a good read to go through before diving into the code.

If we're aiming for consistency across all plugins, feel free to substitute the lines before with your suggestion, no problem!

**Comment by lieut-data** [comment]
> "This plugin contains both a server and web app portion."

This seems a bit wordy for something geared towards developers -- there's a minimum level of knowledge I'd expect coming in, and a working knowledge of plugins (at least from our existing mattermost.dev docs) seems about right. 0/5, but I wouldn't be in any rush to try to unify things here right away.

File: README.md [hunk 15]
```diff
@@ -1,28 +1,143 @@
...
+
+The RHS shows the active stage and a checklist of the tasks to do. You can assign users to the tasks (remember to add them to the channel!), run the predefined slash commands and, of course, you can message your teammates to triage and solve the incident.
+
+Read more about Incidents [here](https://docs.mattermost.com/administration/devops-command-center.html#starting-an-incident).
+
+### Manage your incidents
+
+There will come a time where you will have several incidents, some of them ongoing, some ended. If you ever need a quick glance at the status of an incident, follow these steps:
+
+1. Navigate to Playbooks & Incidents in the main menu.
```
Comment (justinegeffen) [inline README.md:48]:
```suggestion
1. Navigate to **Main Menu > Incidents & Playbooks**.
```

File: README.md [hunk 16]
```diff
@@ -1,28 +1,143 @@
...
+2. Click on Incidents on the header.
```
Comment (justinegeffen) [inline README.md:49]:
```suggestion
2. Select **Incidents** in the header.
```

File: README.md [hunk 17]
```diff
@@ -1,28 +1,143 @@
...
+3. See the list of all incidents. You can filter by Commander, Status (ongoing or ended) and look for a specific incident using the search box.
+4. Click on the incident you are interested in.
```
Comment (justinegeffen) [inline README.md:51]:
```suggestion
4. Select the incident you want to learn more about.
```

File: README.md [hunk 18]
```diff
@@ -1,28 +1,143 @@
...
+5. A summary with some statistics of the incident will show. From this screen, you can also jump to the channel (with the button in the Messages box) and export the channel log, which will download a CSV file with a transcript of all the messages in the channel.
```
Comment (justinegeffen) [inline README.md:52]:
```suggestion
5. The summary screen includes statistics about the incident. You can also jump to the channel as well as export the channel log .CSV file which is a transcript of all the messages in the channel.
```

File: README.md [hunk 19]
```diff
@@ -1,28 +1,143 @@
...
+
+## Contributing
+
+### Prerequisites
+
+There are two things you need to do before starting development on the plugin:
+
+1.  You will need a running server to deploy the plugin to, and the web app to manually test your changes. If you have never set up your Mattermost development environment, do so following the instructions for both the [server](https://developers.mattermost.com/contribute/server/developer-setup/) and the [web app](https://developers.mattermost.com/contribute/webapp/developer-setup/).
```
Comment (justinegeffen) [inline README.md:60]:
```suggestion
1.  You'll need a running server for the plugin deployment, and the web app to manually test your changes. If you have never set up your Mattermost development environment, do so following the instructions for both the [server](https://developers.mattermost.com/contribute/server/developer-setup/) and the [web app](https://developers.mattermost.com/contribute/webapp/developer-setup/).
```

File: README.md [hunk 20]
```diff
@@ -1,28 +1,143 @@
...
+2.  Normally, the plugin needs a Mattermost E20 License to start. However, you can bypass this and run it for development by configuring your Mattermost server to enable both testing and developer mode. You can do so directly on the server's `config/config.json` file, setting both `ServiceSettings.EnableTesting` and `ServiceSettings.EnableDeveloper` to `true` (don't forget to restart the server afterwards). You can also configure these settings via the System Console, under Environment > Developer. See [License](#License) for more information.
```
Comment (justinegeffen) [inline README.md:61]:
```suggestion
2.  Normally, the plugin needs a Mattermost Enterprise Edition E20 License to start. However, you can bypass this and run it for development by configuring your Mattermost server to enable both testing and developer mode. You can do so directly on the server's `config/config.json` file, setting both `ServiceSettings.EnableTesting` and `ServiceSettings.EnableDeveloper` to `true` (don't forget to restart the server afterwards). You can also configure these settings via the System Console, under **Environment > Developer**. See [License](#License) for more information.
```

File: README.md [hunk 21]
```diff
@@ -1,28 +1,143 @@
...
+
+If this is the first time you contribute to a Mattermost plugin, the [plugin documentation](https://developers.mattermost.com/extend/plugins/) is a good read to go through before diving into the code.
```
Comment (justinegeffen) [inline README.md:63]:
```suggestion
If this is the first time you're contributing to a Mattermost plugin, the [plugin documentation](https://developers.mattermost.com/extend/plugins/) is a good read to go through before diving into the code.
```

File: README.md [hunk 22]
```diff
@@ -1,28 +1,143 @@
...
+You have a running server and web app, and you have read the plugin documentation. Now you only need to do the usual dance: `clone`, `cd`, `make`.
+
+```sh
+# Change the URL below to your fork if you plan to make a PR!
+git clone git@github.com:mattermost/mattermost-plugin-incident-response.git
+cd mattermost-plugin-incident-response
+make
+```
+
+The default target for `make` triggers the commands `check-style`, `test` and `dist`: if everything works as expected, you will end up with a packaged plugin in the `dist/` directory, ready to be uploaded to Mattermost via the System Console. There are more `make` commands: run `make help` to get a comprehensive list of them.
```
Comment (justinegeffen) [inline README.md:76]:
```suggestion
The default target for `make` triggers the commands `check-style`, `test`, and `dist`: if everything works as expected, you'll end up with a packaged plugin in the `dist/` directory, ready to be uploaded to Mattermost via the System Console. There are more `make` commands: run `make help` to get a comprehensive list of them.
```

File: README.md [hunk 23]
```diff
@@ -1,28 +1,143 @@
...
+
+One particularly interesting command is `make deploy`, which looks for a running Mattermost server, and directly deploys the plugin to it and enables it via the API. For this to work, you need to set up a few environment variables before:
```
Comment (justinegeffen) [inline README.md:78]:
```suggestion
One particularly interesting command is `make deploy`, which looks for a running Mattermost server, and directly deploys the plugin to it and enables it via the API. For this to work, you first need to set up a few environment variables:
```

File: README.md [hunk 24]
```diff
@@ -1,28 +1,143 @@
...
+
+# You can now deploy the plugin from the comfort of your terminal!
+make deploy
+```
+
+### Testing
+
+#### Unit tests
+
+Running the unit tests for both the server and the webapp is as easy as running
```
Comment (justinegeffen) [inline README.md:98]:
```suggestion
Running the unit tests for both the server and the web app is as easy as running:
```

File: README.md [hunk 25]
```diff
@@ -1,28 +1,143 @@
...
+
+```sh
+make test
+```
+
+Please note that the tests of the [`sqlstore`](server/sqlstore) package need a running server to work: they create temporal databases using the PostgreSQL and MySQL containers created by the server.
+
+If you want to run only the tests of the server part, you may do so with the usual `go test` command, as in
```
Comment (justinegeffen) [inline README.md:106]:
```suggestion
If you want to run only the tests of the server part, you may do so with the usual `go test` command, as in:
```

**Review by justinegeffen — APPROVED** [review]

File: README.md (hunk 16 above)
Comment (agarciamontoro) [reply README.md:51]: I *never* know if things are on the header or in the header or at the channel or in the... Thank you for this!

File: README.md (hunk 17 above)
Comment (agarciamontoro) [reply README.md:53]: This is much better, what a difference a couple of words can make!

**Comment by agarciamontoro** [comment]
Thank you for the thorough review, @justinegeffen! It's amazing what some well made adjustments can do to improve a text :pray: I'll merge this as soon as CI passes! :tada:
